src/testing/iotest/reader.go

```diff
@@ -86,3 +87,17 @@ func (r *timeoutReader) Read(p []byte) (int, error) {
 	}
 	return r.r.Read(p)
 }
+
+// ErrIO is a fake IO error
```

Comment:
doc comments should end with `.`s:
```suggestion
// ErrIO is a fake IO error.
```